project-management/src/test/groovy/life/qbic/projectmanagement/application/AddExperimentToProjectServiceSpec.groovy

```diff
@@ -45,7 +45,7 @@ class AddExperimentToProjectServiceSpec extends Specification {
                 ProjectTitle.of("Oral microbiome study"),
                 ProjectObjective.create("Analysis if tooth paste has an impact oral health and the mouth microbiome"))
         ProjectCode projectCode = ProjectCode.random()
-        Contact personReference = new Contact("John Doe", "john@doe.abcdefg")
+        Contact personReference = new Contact("John Doe", "john@doe.abcdefg", "", "")
```

Comment:
The issue with empty values vs non-existing values. (See my comments in the Contact class)